rp/frontend/static/.well-known/ii-alternative-origins

```diff
@@ -1,6 +1,6 @@
 {
   "alternativeOrigins": [
-    RP_FE_HOSTNAME_PLACEHOLDER
+    "http://localhost:5173",
```

Comment:
When removing `RP_FE_HOSTNAME_PLACEHOLDER` one should update also places where it is used, namely `scripts/deploy-rp.sh` (ll. 112 and 116 in master, ll. 124 and 128 in this PR below)... 
Or is it just a leftover-change from debugging? 🤔

Reply:
oi, this is a leftover change, thanks!!